services-speech/src/main/java/com/mapbox/api/speech/v1/MapboxSpeech.java

```diff
@@ -0,0 +1,178 @@
+package com.mapbox.api.speech.v1;
+
+
+import com.google.auto.value.AutoValue;
+import com.mapbox.core.exceptions.ServicesException;
+import com.sun.istack.internal.Nullable;
+
+import java.io.File;
+
+import okhttp3.Cache;
+import okhttp3.OkHttpClient;
+import okhttp3.ResponseBody;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+
+import static com.mapbox.core.constants.Constants.BASE_API_URL;
+
+@AutoValue
+public abstract class MapboxSpeech {
+  /**
+   * Makes the API call to get the instruction using parameters specified when MapboxSpeech was
+   * built.
+   *
+   * @param instruction text to pass to API voice for dictation
+   * @param callback to alert status updates for API call
+   */
+  public void getInstruction(String instruction, Callback<ResponseBody> callback) {
```

Comment:
Love this convenience method here. It goes against our other APIs however, could we include, in addition to this method, `executeCall` and `enqueueCall`.